ddl/ddl_api.go

```diff
@@ -498,6 +498,10 @@ func setEmptyConstraintName(namesMap map[string]bool, constr *ast.Constraint, fo
 		colName := constr.Keys[0].Column.Name.L
 		constrName := colName
 		i := 2
+		if strings.ToUpper(constrName) == mysql.PrimaryKeyName {
```

Comment:
Because MySQL will append `_{num}` to anonymous index name if index column name is `primary`.